bokehjs/test/integration/glyphs/markers.ts

```diff
@@ -112,4 +113,39 @@ describe("Marker glyph", () => {
 
     await display(row([p0, p1, p2]))
   })
+
+  it("should support hatch", async () => {
+    const hatch_patterns: HatchPattern[] = [
+      "dot", "ring", "horizontal_line", "vertical_line", "cross", "horizontal_dash",
+      "vertical_dash", "spiral", "right_diagonal_line", "left_diagonal_line", "diagonal_cross",
+      "right_diagonal_dash", "left_diagonal_dash", "horizontal_wave", "vertical_wave",
+      "criss_cross", "dot", "ring", "horizontal_line", "vertical_line", "cross",
+    ]
+
+    const markers: MarkerType[] = [
+      "circle", "circle_cross", "circle_dot", "circle_x", "circle_y", "diamond", "diamond_cross",
+      "diamond_dot", "hex", "hex_dot", "inverted_triangle", "square", "square_cross", "square_dot",
+      "square_pin", "square_x", "star", "star_dot", "triangle", "triangle_dot", "triangle_pin",
+    ]
```

Comment:
This should be imported from `core/enums` and then use e.g. `MarkerType.values`.

Reply:
The test only uses a subset of all the marker types as some do not have any fill and therefore do not have any hatch (e.g. `asterisk`, `dash`, `x`, etc). If I used `MarkerType.values` I would have to filter them anyway, so I either have to name the ones I do want or the ones I don't want.

Reply:
I see. Well, we could provide enums of finer granularity in both cases, but that can be left for future PRs.